lib/bulbs-elements/util/dimensions.js

```diff
@@ -38,6 +38,26 @@ export function getScrollOffset () {
   return { x, y };
 }
 
+export function getRoundedBoundingClientRect (element) {
```

Comment:
Need this utility to get around firefox having too much precision with these pixel values. Prevents problems trying to figure out when bounding box values are equal.